src/coreclr/src/gc/gc.cpp

```diff
@@ -20249,6 +20321,25 @@ size_t gc_heap::get_total_gen_estimated_reclaim (int gen_number)
     return total_estimated_reclaim;
 }
 
+#ifdef VERIFY_COMMITTED_BY_OH
+void gc_heap::verify_committed_by_oh_per_heap(heap_segment* inst)
+{
+    gc_oh_num oh = heap_segment_oh (inst);
+    generation* gen = generation_of (oh + max_generation);
+    heap_segment* seg = heap_segment_rw (generation_start_segment (gen));        
+    size_t verify = 0;
```

Comment:
>size_t verify = 0; [](start = 4, length = 18)

nit - I would probably call this something like recorded_committed instead of "verify"....